### ISSUE TITLE
Please add support for UserScripts

### ISSUE THREAD

**Issue by ale82to** [SoulBrowser/SoulBrowser#8]
as title says please consider adding supports for userscripts! in yuzu and berry browsers work perfectly and are webview based.

**Comment by lalalasupa0** [comment]
This should be closed lol

**Comment by KaKi87** [comment]
Hi,

I don't know if that was the case at the time you created this issue @ale82to, but Soul Browser does support userscripts today.

Though if you encounter problems when using them, feel free to report them, otherwise I invite you to please close this issue if my answer is satisfactory.

Thank you

---

I'm maintaining a modded version of Soul Browser with several bugfixes and added features, with the source code published [here](https://git.kaki87.net/VibedByKaKi/soul-browser).

**Comment by lalalasupa0** [comment]
> Hi,
> 
> I don't know if that was the case at the time you created this issue [@ale82to](https://github.com/ale82to), but Soul Browser does support userscripts today.
> 
> Though if you encounter problems when using them, feel free to report them, otherwise I invite you to please close this issue if my answer is satisfactory.
> 
> Thank you
> 
> I'm maintaining a modded version of Soul Browser with several bugfixes and added features, with the source code published [here](https://git.kaki87.net/VibedByKaKi/soul-browser).

Where is the download in that git? I only see Obtanium, but that fails. Apk?

**Comment by KaKi87** [comment]
Yeah I made a small mistake regarding the tagging of the initial release which makes Obtainium fail but if I manually fix it from time to time, which I just did, it should work again if you'd like to try now.

Otherwise, you can of course manually download APKs in the [Releases](https://git.kaki87.net/VibedByKaKi/soul-browser/releases) tab, which is, after all, what Obtainium is configured to do.
